chocolatey/ChocolateyGUI
# (build) Refactors embedded WiX package

## Description Of Changes
This PR removes the WiX tool files, and updates the path in ChocolateyGui.Install to the relative path that will be present when Chocolatey Cake Recipe loads the tool for the Build-MSI step.

Requires recipe changes introduced by Chocolatey Cake Recipe 0.21.0.

## Motivation and Context
We recently standardised another MSI build to use Cake, and Chocolatey Cake Recipe. We decided not to embed the WiX files in that repository, but instead to install the tool when it was needed.

This brings consistency to the Chocolatey GUI MSI creation.

## Testing
1. Checkout branch
2. Run `.\build.ps1`  
    Please note that on first run, you'll get a quick error during the Restore step - everything still works, though, and the error isn't present after that first run (as WiX is restored by the `Build-MSI` step)
    ![image](https://user-images.githubusercontent.com/1975761/227179916-f61cb082-e080-492a-8537-77f16a86c123.png)
3. Observe that MSI is produced, build is successful, `.\tools\WiX.3.11.2\` exists

### Operating Systems Testing
- Windows 10

## Change Types Made

## Change Checklist
